tests/test_fourier.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from iospi import fourier
+
+
+# right way to use as global in testing environment?
+N = 128
+arr2d = np.random.normal(size=N * N).reshape(N, N)
+arr3d = np.random.normal(size=N * N * N).reshape(N, N, N)
+
+
+def test_do_fft():
+    """Test fft wrapper.
+
+    Conservation of variance due to gaussian white noise input.
+    """
+    for arr, d in zip([arr2d, arr3d], [2, 3]):
+        arr_f = fourier.do_fft(arr, d=d)
+        assert arr_f.dtype == "complex128"
+        assert arr_f.shape == tuple([N] * d)
+        assert np.isclose(
+            np.var(arr), np.var(arr_f), atol=1e-3
+        )  # change atol if N different
+
+        assert np.allclose(
+            fourier.do_fft(arr, d=d, only_real=True),
+            fourier.do_fft(arr, d=d).real,
+        )
+
+
+def test_do_ifft():
+    """Test ifft wrapper."""
+    for arr, d in zip([arr2d, arr3d], [2, 3]):
+        arr_i = fourier.do_ifft(arr, d=d)
+        assert arr_i.dtype == "float64"
+        assert arr_i.shape == tuple([N] * d)
+
+        # needs work: normalization constant
```

Comment:
If the function is not ready then it shouldnt be in the codebase

Reply:
I think it's fine, I'll just remove the codebase. The important thing is to test the fft and ifft together and make sure the same function is recovered, which I did.